data/json/recipes/armor/other.json

```diff
@@ -13,6 +13,45 @@
     "components": [ [ [ "rag", 35 ] ] ]
   },
   {
+    "result": "ring",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_OTHER",
+    "skill_used": "fabrication",
+    "difficulty": 6,
+    "time": 120000,
+    "book_learn": [ [ "jewelry_book", 6 ] ],
+    "qualities": [
+      { "id": "HAMMER", "level": 3 },
+      { "id": "CHISEL", "level": 3 }
+    ],
+    "tools": [
+      [ [ "tongs", -1 ] ],
+      [ [ "anvil", -1 ] ],
+      [ [ "swage", -1 ] ],
+      [ [ "forge", 50 ], [ "oxy_torch", 10 ] ]
+    ],
+    "components": [ [ [ "gold_small", 3 ] ] ]
+  },
+  { 
```

Comment:
There is an excessive space after the bracket.
You should run your files through the linting tool (http://dev.narc.ro/cataclysm/format.html).

Reply:
Thanks!